presidio-analyzer/Dockerfile.scai-all

```
@@ -0,0 +1,53 @@
+ENV WORKERS=1
+
+WORKDIR /app
+
+# Install essential build tools and curl for health checks
+RUN apt-get update \
+  && apt-get install curl --no-install-recommends -y \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY ./pyproject.toml /app/
+
+RUN pip install poetry \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)